components/feature/pwa/build.gradle

```diff
@@ -12,6 +12,7 @@ android {
     defaultConfig {
         minSdkVersion config.minSdkVersion
         targetSdkVersion config.targetSdkVersion
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
```

Comment:
Is this necessary? Most of our tests specify the runner with `RunWith`

Reply:
I'm actually not sure. Will check it out.

Reply:
It seems I can't use `RunWith`: `Type mismatch: inferred type is KClass<AndroidJUnitRunner> but KClass<out Runner!> was expected`. 

I borrowed this from https://github.com/mozilla-mobile/android-components/blob/f7d2a4648162bebc98f852e2f2f9d6bd8b6f0636/components/browser/engine-gecko-nightly/build.gradle#L30 where it's also done like this.